packages/itwin/tree-widget/src/test/trees/VisibilityTreeEventHandler.test.ts

```diff
@@ -167,4 +167,28 @@ describe("VisibilityTreeEventHandler", () => {
     });
 
   });
+
+  it("getNodeCheckBoxInfo doesn't call getVisibilityStatus when visibilityHandler is undefined", async () => {
```

Comment:
I think we can update `VisibilityTreeEventHandler` API and make `VisibilityHandler` required.

Reply:
fixed